Makefile

```diff
@@ -1374,6 +1388,31 @@ mpi_f08_test:
 	$(if $(findstring 1,$(MPAS_MPI_F08)), $(eval MPI_F08_MESSAGE = "Using the mpi_f08 module."), )
 	$(if $(findstring 1,$(MPAS_MPI_F08)), $(info mpi_f08 module detected.))
 
+musica_fortran_test:
+	@#
+	@# Create a Fortran test program that will link against the MUSICA library
+	@#
+	$(info Building a simple program with the MUSICA-Fortran library...)
```

Comment:
Just for consideration, we could say something like `Checking for a working MUSICA-Fortran library...`, as that's really the point of this build target. How that's done -- whether through the use of a simple program or otherwise -- probably matters less to users.